components/src/core/components/Input/date-input.scss

```diff
@@ -30,6 +30,29 @@
 .oxd-date-input {
   display: flex;
   align-items: center;
+  position: relative;
+
+  // Inline label styling
+  &--inline-label {
+    position: absolute;
+    top: $oxd-inline-label-top;
+    left: $oxd-inline-label-left;
+    font-size: $oxd-inline-label-font-size;
+    color: $oxd-input-placeholder-color;
+    pointer-events: none;
```

Comment:
This style will cause any content added inside this not to respond to click events. Hope that's what you intended